CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Unreleased
+# 2.8.3
```

Comment:
Minor: I'd probably say this was a minor change rather than a patch as per -> https://semver.org/

Reply:
+1